addons/sale/migrations/9.0.1.0/pre-migration.py

```diff
@@ -1,36 +1,41 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenUpgrade module for Odoo
-#    @copyright 2014-Today: Odoo Community Association
-#    @author: Sylvain LE GAL <https://twitter.com/legalsylvain>
```

Comment:
Removing copyrights on an existing file is a death sin. Note that original author was @coleste who put in the wrong copyrights